Functions.Templates/Templates/DaprTopicTrigger-PowerShell/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "defaultFunctionName": "DaprTopicTrigger",
+  "description": "$DaprTopicTrigger_description",
```

Comment:
Please add a matching description string to the resource file located at /azure-functions-templates/Functions.Templates/Resources/Resources.resx